utils/chsmack.c

```diff
@@ -148,6 +148,59 @@ static void modify_transmute(const char *path)
 	}
 }
 
+/* modify the file (or directory) of path */
+static void modify_file(const char *path)
+{
+	modify_prop(path, &access_set, XATTR_NAME_SMACK);
+	modify_prop(path, &exec_set, XATTR_NAME_SMACKEXEC);
+	modify_prop(path, &mmap_set, XATTR_NAME_SMACKMMAP);
+	modify_transmute(path);
+}
+
+/* print the file (or directory) of path */
+static void print_file(const char *path)
+{
+	int rc;
+	char *label;
+
+	/* Print file path. */
+	printf("%s", path);
+
+	errno = 0;
+	rc = (int)smack_new_label_from_path(path,
```

Comment:
Is downcasting from `ssize_t` to `int` really necessary? Since all `smack_new_label_*` functions return `ssize_t`, changing type of `rc` to `ssize_t` would be better.